monai/networks/utils.py

```diff
@@ -654,32 +663,38 @@ def convert_to_onnx(
                 del kwargs["example_outputs"]
             mode_to_export = torch.jit.script(model, **kwargs)
 
+        if torch.is_tensor(inputs) or isinstance(inputs, dict):
+            onnx_inputs = (inputs,)
+        else:
+            onnx_inputs = tuple(inputs)
+
         if filename is None:
             f = io.BytesIO()
-            torch.onnx.export(
-                mode_to_export,
-                tuple(inputs),
-                f=f,
-                input_names=input_names,
-                output_names=output_names,
-                dynamic_axes=dynamic_axes,
-                opset_version=opset_version,
-                **torch_versioned_kwargs,
-            )
+        else:
+            f = filename
+
+        torch.onnx.export(
+            mode_to_export,
+            onnx_inputs,
+            f=f,
+            input_names=input_names,
+            output_names=output_names,
+            dynamic_axes=dynamic_axes,
+            opset_version=opset_version,
+            do_constant_folding=do_constant_folding,
+            # dynamo=dynamo,
```

Comment:
```suggestion
```

Reply:
Actually, now that we are upgraded to 24.08, it should be propagated - otherwise 'onnx_dynamo' option will not work.